data/remnant missions.txt

```diff
@@ -495,3 +560,718 @@ mission "Remnant: Technology Available"
+			`	By the time you finish releasing the eggs back into the clouds, several void sprites have converged on your ship and appear to be herding the eggs away from you with gentle buffets of their wings. You take a moment to appreciate the alien beauty of these creatures before closing the hatch and purging the ship with fresh air.`
+	on complete
+		"reputation:  Drak " = 1
+		conversation
+			`Your return trip to Aventine is just as exciting, dodging the Archon on the way out of the system.`
+			`	Back on the ground, you report on how the release went. The researcher is pleased to hear that the void sprites responded to their presence. "We will give them a few weeks, then try sending another ship to monitor them. Hopefully the Archon will be pacified by this. Meet me in the alien environments lab just off the spaceport if you are interested in pursuing this."`
+
+
+
+mission "Remnant: Return the Samples 2"
+	name "Return the Samples"
+	description "A researcher on <planet> has asked you to return some more samples to Slylandro."
+	source Aventine
+	stopover Slylandro
+	to offer
+		has "Remnant: Return the Samples: done"
+	on offer
+		conversation
+			`Following Plume's directions, you find a lab tucked behind the main area of the spaceport, where a short walkway leads into an underground structure. Stepping inside the door you note a rack of camouflage netting positioned easily at hand. Given what you have learned of their history, it is probably a last ditch preparation in case they need to hide the entrance from a raid.`
+			`	The lab itself is a microcosm of Remnant development, the structure appears to be some strange semi-organic material, and you can see an eclectic mix of familiar and unfamiliar shapes cluttering the tables. If you had to guess, you would think that this lab was the result of several different families of technology, only one of which was human.`
+			`	Plume and several other Remnant are at the back of the room inspecting a bunch of cocoon-like containers, similar to the ones that carried the last batch of eggs back to Nasqueron. After checking a screen he looks up and seems momentarily startled to see you.`
+			`	"Ah, <first>! You are just in time," Plume chants quickly, his hands unconsciously dancing in a quick series of gestures. "These cocoons are ready to be delivered, this time to Slylandro. Could you drop them off like you did before?"`
+			choice
+				`	"Yes, I would be happy to bring them home."`
+				`	"Sorry, I'm really not interested in braving the Archon a third time."`
```

Comment:
I'd change the wording a bit here so that there aren't two decline choices in a row that you the phrase "braving the Archon".

Reply:
Now they are braving, dodging, and risking, respectively.